src/lib/atoms/performance/atom-family.ts

```diff
@@ -25,8 +25,8 @@ import { createAtomFamilyWithCleanup } from './cleanup'
  * // Clean up when done: workoutAtomFamily.remove('workout-123')
  * ```
  */
-export const workoutAtomFamily = atomFamily((workoutId: string) =>
-  atom(
+export const workoutAtomFamily = atomFamily((workoutId: string) => {
+  const a = atom(
```

Comment:
@CharlieHelps - there's got to be a better way to do this right? 
can't we add a debug label at instantiation?